products/BellHybrid/apps/application-bell-main/presenters/HomeScreenPresenter.cpp

```diff
@@ -245,4 +246,16 @@ namespace app::home_screen
         app->switchWindow(gui::BellBatteryStatusWindow::name,
                           std::make_unique<gui::BellBatteryStatusWindow::Data>(getBatteryLvl(), isBatteryCharging()));
     }
+
+    UTF8 HomeScreenPresenter::getGreeting()
+    {
+        const auto greetingCollection = utils::translate_array("app_bell_greeting_msg");
+        if (greetingCollection.empty()) {
+            LOG_WARN("app_bell_greeting_msg array does not exist, using default string");
+            return "app_bell_greeting_msg";
```

Comment:
Shouldn't it return some default greeting, not "app_bell_greeting_msg"?

Reply:
This is the situation when our greetingCollection is empty - that means we have no greetings in current language. Displaying the string identifier is coherent with the rest of the system - if any string is missing in current translation - then we display the string identifier.

Do you suggest hardcoding some greeting in the code? I believe that would mean an English default greeting for any current language